### PR TITLE
Use same format as in mix.exs generated with Elixir v1.7 for dependen…

### PR DESCRIPTION
**Pull request by eksperimental**
…cies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,15 +19,19 @@ If you are using Elixir v1.7 and later:
 
 ```elixir
 def deps do
-  [{:ex_doc, "~> 0.19", only: :dev, runtime: false}]
+  [
+    {:ex_doc, "~> 0.19", only: :dev, runtime: false},
```
Comment (fertapric) [inline README.md]:
```suggestion
    {:ex_doc, "~> 0.19", only: :dev, runtime: false}
```

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by josevalim** [comment]
> It breaks the rules of the formatter, but thia rule is still under discussion.

FWIW, It is not under discussion. We don't do trailing commas. :)

**Comment by eksperimental** [comment]
I read one PR where you mentioned that we were waiting until v1.8 to give more time for people to use  it and get more feedback, and then make a decission about it

**Pull request merged**